### DISCUSSION TITLE
Set of files compatible with RHEL

### DISCUSSION THREAD

**Discussion by MrAda** [alire-project/alire#1726] [General]
Does anyone have a list of what versions of gnatstudio, gprbuild, gnat compiler, etc go with a particular version of RHEL?
I can use the community version on 7.9 without issue but I need to upgrade to the latest and greatest that will be supported.
I tried to do something similar with 8.4 of RHEL and had a lot of issues.  A list would help me immensely.

**Comment by MrAda** [comment]
This is kind of related to 1689 issue but yet I have gone back to RHEL 7.9.  In my case I tried using Alr and it complained of GLIBC 2.28 and I am on 2.17.  I don't feel comfortable trying to upgrade GLIBC.  So I am guessing there really isn't a good build strategy using Alire for RHELs in general. 
So I noticed that I don't necessarily need to use alr to build, just that I need my compilation environment setup.  Maybe I can somehow take my 2021 community version and upgrade it.  I know I can upgrade the gnatstudio to the latest without issue.   Then I would have to upgrade all of the tools and gprbuilds, etc.  Has someone done this and what is the best way to tackle this?

**Reply by mosteo** [reply]
If you only want more recent GNATs+Gprbuild, you could check if [our AppImage](https://github.com/alire-project/alire/releases/download/v2.0.1/alr-2.0.1-x86_64.AppImage) works, as it is built on a somewhat older system (Ubuntu 20.04). If so, you could then use it to just download the toolchains. But I'm unsure if these toolchains will have similar GLIBC issues.

Perhaps you can check the build scripts at https://github.com/alire-project/GNAT-FSF-builds and try to build on your platform.

**Reply by MrAda** [reply]
I have an air gapped system so using alire is not doable for downloading stuff
